src/api/docker-files/Dockerfile

```diff
@@ -17,6 +17,9 @@ ADD Gemfile /obs/src/api/Gemfile
 ADD Gemfile.lock /obs/src/api/Gemfile.lock
 RUN chown -R frontend /obs/src/api
 
+# Install tar for building gem `gpgme`
+RUN zypper install -y tar
```

Comment:
Please add this to our [bootstrap script](https://github.com/openSUSE/open-build-service/blob/master/docker-files/base/docker-bootstrap.sh#L6) instead

Reply:
@hennevogel 

> Please add this to our bootstrap script instead

Done.